react-table/src/Paging.tsx

```diff
@@ -32,8 +32,12 @@ export default function (props: IProps) {
     const [pages, setPages] = React.useState<number[]>([]);
 
     const nPages = 7;
-    const minPg = React.useMemo(() => Math.min(...pages), [pages]);
-    const maxPg = React.useMemo(() => Math.max(...pages), [pages]);
+    const minPg = React.useMemo(() => {
+        const min = Math.min(...pages);
+        if (isFinite(min)) return min;
```

Comment:
```suggestion
        if (isFinite(min) && !isNaN(min)) return min;
```
Just to cover all bases